src/ACEtk/DosimetryTable/test/DosimetryTable.test.cpp

```diff
@@ -0,0 +1,505 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "ACEtk/fromFile.hpp"
+#include "ACEtk/DosimetryTable.hpp"
+
+// other includes
+
+// convenience typedefs
+using namespace njoy::ACEtk;
+using DosimetryCrossSectionData = block::DosimetryCrossSectionData;
+using MTR = block::MTR;
+using SIGD = block::SIGD;
+using Header = Table::Header;
+
+void verifyChunk( const DosimetryTable& );
+
+SCENARIO( "DosimetryTable" ){
+
+  GIVEN( "valid data for a DosimetryTable" ) {
+
+    auto table = fromFile( "13027.24y" );
+    std::array< int32_t, 16 > iz = table.data().IZ();
```

Comment:
[Design suggestion - future]
Should `iz_size`, etc, be constants, or even have `using iz_array = std::array<int32_t, 16>`?